bot/kodiak/test_evaluation.py

```diff
@@ -257,6 +273,29 @@ def pull_request() -> PullRequest:
     )
 
 
+def create_pull_request() -> PullRequest:
```

Comment:
All of the pytest fixtures will be removed once everything has been migrated to the `create_`-prefixed functions.